<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormIncluirApagar</class>
 <widget class="QWidget" name="FormIncluirApagar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1307</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>APAgar</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{text-transform: uppercase;
background-color: rgb(207, 207, 207);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">

QWidget{
text-transform: uppercase;

	background-color: rgb(85, 170, 255,150);
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget#widget_2{
border:none;
	background-color: rgb(85, 170, 255,0);
}
</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="minimumSize">
            <size>
             <width>980</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>950</width>
             <height>700</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">

QFrame{
text-transform: uppercase;
border:1px solid blue;
	background-color: rgb(85, 170, 255,150);
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QFrame" name="frame">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>80</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QWidget{border: none;
background-color: qlineargradient(spread:pad, x1:0.561, y1:0.591364, x2:0.558, y2:0.914773, stop:0 rgba(0, 188, 255, 255), stop:1 rgba(201, 255, 255, 255));}

QLabel{
background:none;
font-size: 25px;
font-family: &quot;Arial&quot;;
font-weight: bold;
	color: black;
border: none;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label">
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>incluir apagar</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="styleSheet">
               <string notr="true">QFrame
{background: #FFF;
border: none
}

QLineEdit{
background-color: rgb(255, 255, 127);
font-size: 16px;
font-family: &quot;Arial&quot;;
font-weight: bold;
color: rgb(0, 0, 0);
border:none;
border-bottom-left-radius:0px;
border-bottom-right-radius:0px;
border-top-left-radius:0px;
padding:6px
}
QLineEdit:focus{
	background-color: rgb(226, 226, 226);
}

QPushButton{
	text-align:center;
	color:#fff;
	border:0px;
	
	font-size:20px;
	cursor: pointer;
	background-color: rgb(0, 85, 255);
padding:15px 15px;
}
QPushButton:hover{
background-color:#3668ff;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <widget class="QLineEdit" name="tx_descricao">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>30</y>
                 <width>281</width>
                 <height>41</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>41</height>
                </size>
               </property>
              </widget>
              <widget class="QLabel" name="label_2">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>10</y>
                 <width>111</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>descriçao</string>
               </property>
              </widget>
              <widget class="QFrame" name="frame_5">
               <property name="geometry">
                <rect>
                 <x>360</x>
                 <y>10</y>
                 <width>231</width>
                 <height>71</height>
                </rect>
               </property>
               <property name="maximumSize">
                <size>
                 <width>500</width>
                 <height>80</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QDateEdit {
	border: 2px solid lightgray;
	border-radius: 5px;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
	background-color: rgb(255, 255, 255);
}
 
</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <property name="spacing">
                 <number>8</number>
                </property>
                <property name="leftMargin">
                 <number>8</number>
                </property>
                <property name="topMargin">
                 <number>8</number>
                </property>
                <property name="rightMargin">
                 <number>8</number>
                </property>
                <property name="bottomMargin">
                 <number>8</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>lançamento entre data</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_7">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QDateEdit" name="dt_inicial">
                     <property name="focusPolicy">
                      <enum>Qt::ClickFocus</enum>
                     </property>
                     <property name="styleSheet">
                      <string notr="true"/>
                     </property>
                     <property name="calendarPopup">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QLabel" name="label_6">
               <property name="geometry">
                <rect>
                 <x>30</x>
                 <y>90</y>
                 <width>121</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>cod fornec (F11)</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="tx_codigoFornecedor">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>110</y>
                 <width>121</width>
                 <height>41</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>41</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="contextMenuPolicy">
                <enum>Qt::ActionsContextMenu</enum>
               </property>
               <property name="placeholderText">
                <string>cod fonecedor</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="tx_descricaofornecedo">
               <property name="geometry">
                <rect>
                 <x>150</x>
                 <y>110</y>
                 <width>281</width>
                 <height>41</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>41</height>
                </size>
               </property>
              </widget>
              <widget class="QLabel" name="label_7">
               <property name="geometry">
                <rect>
                 <x>150</x>
                 <y>90</y>
                 <width>251</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>descriçao fornecedor</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="tx_valortotalparcela">
               <property name="geometry">
                <rect>
                 <x>460</x>
                 <y>110</y>
                 <width>151</width>
                 <height>41</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>41</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">
</string>
               </property>
               <property name="text">
                <string>0,00</string>
               </property>
               <property name="maxLength">
                <number>3275</number>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
              <widget class="QLabel" name="label_8">
               <property name="geometry">
                <rect>
                 <x>470</x>
                 <y>90</y>
                 <width>141</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>valor</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="tx_quantidadeparcela">
               <property name="geometry">
                <rect>
                 <x>630</x>
                 <y>110</y>
                 <width>171</width>
                 <height>41</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>41</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>MS Shell Dlg 2</family>
                 <pointsize>11</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
               </property>
               <property name="text">
                <string>1</string>
               </property>
               <property name="maxLength">
                <number>3</number>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
              <widget class="QLabel" name="label_9">
               <property name="geometry">
                <rect>
                 <x>630</x>
                 <y>90</y>
                 <width>171</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>Quantidade parcela</string>
               </property>
              </widget>
              <widget class="QTableWidget" name="tab_contApagar">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>160</y>
                 <width>911</width>
                 <height>351</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="contextMenuPolicy">
                <enum>Qt::ActionsContextMenu</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">QTableView{
background: #F1F1F1;
color: black;
font-weight: bold;
font-size: 16px;
border:1px solid;
selection-color: rgb(0, 0, 255);
}
QHeaderView:section{
background: #FFF;

font-size: 13px;
font-family: &quot;Arial&quot;;
font-weight: bold;
color: #797979;
border:none;
border-bottom: 2px solid black;
border-top:2px solid black;
padding:5px 0px;
text-transform: uppercase
}</string>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="tabKeyNavigation">
                <bool>false</bool>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>150</number>
               </attribute>
               <attribute name="horizontalHeaderHighlightSections">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderCascadingSectionResizes">
                <bool>false</bool>
               </attribute>
               <column>
                <property name="text">
                 <string>descriçao</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>parcelas</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>R$ valor</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>data lançamento</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>data vencimento</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>fornecedor</string>
                </property>
               </column>
              </widget>
              <widget class="QPushButton" name="bt_proceSPagamento">
               <property name="geometry">
                <rect>
                 <x>660</x>
                 <y>10</y>
                 <width>291</width>
                 <height>61</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>20</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>processar pagamento</string>
               </property>
              </widget>
              <widget class="QPushButton" name="bt_adicionar">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>520</y>
                 <width>191</width>
                 <height>61</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>20</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>adicionar</string>
               </property>
              </widget>
              <widget class="QPushButton" name="bt_cancelar">
               <property name="geometry">
                <rect>
                 <x>750</x>
                 <y>520</y>
                 <width>191</width>
                 <height>61</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>20</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	text-align:center;
	color:#fff;
	border:0px;
	
	font-size:20px;
	cursor: pointer;
	background-color:red;
padding:15px 15px;
}
QPushButton:hover{
background-color:#3668ff;
}
</string>
               </property>
               <property name="text">
                <string>canceclar</string>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionbuscar_fornecedor">
   <property name="icon">
    <iconset resource="../assets/file.qrc">
     <normaloff>:/telainicial/telainicial/lupa.png</normaloff>:/telainicial/telainicial/lupa.png</iconset>
   </property>
   <property name="text">
    <string>buscar fornecedor</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../assets/file.qrc"/>
 </resources>
 <connections/>
</ui>
